src/index.js

```diff
@@ -1,45 +1,54 @@
 import './style.css';
 
-const Tasks = [];
-let count = 0;
+// const updateLocalStorage = require('./storage.js');
```

Comment:
- [ ] Kindly remove all commented codes in your files. It makes your code unreadable and a little messy. :pray:

Reply:
This issue has been well addressed. Great job! :pray: